src/deploy/functions/release.ts

```diff
@@ -94,7 +105,20 @@ export async function release(context: any, options: any, payload: any) {
   // Wait for the first function in each region to be deployed, and all the other calls to be queued,
   // then close the queue.
   // Wait for all of the deployments to complete.
-  await Promise.all(queuePromises);
+  try {
+    await Promise.all(queuePromises);
+  } catch (err) {
+    utils.reject(
```

Comment:
Is this error always going to be encountered because of large function batches? Can we detect whether quota errors specifically were the cause of the rejection?

Also wonder about literally forking the error message based on the number of functions deployed...

Reply:
This will only ever throw if we exceed the max retries on a task, and we only retry quota errors. 

 I don't see any quotas that we're likely to exceed that aren't related to large function deployments, but it might still be worth forking the error message at 50? functions or so.